mobify/progressive-web-scaffold
# Replace images with more optimized images

Replace some images with better optimized versions of themselves. For example, replacing PNG images with JPG versions where it makes sense.

## Changes
- Replace images with more optimized version of themselves

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Observe the images in the homepage carousel: ensure they appear good, and that their file size is in the 30-50kb range instead of 700kb
